modules/creating-a-project-using-the-CLI.adoc

```diff
@@ -7,6 +7,11 @@
 
 If allowed by your cluster administrator, you can create a new project.
 
+[NOTE]
+====
+Projects starting with `openshift-` and `kube-` are considered critical by {product-title}. As such, {product-title} does not allow you to create Projects starting with `openshift-` or `kube-` using the CLI.
```

Comment:
s/using the CLI/using the `oc new-project` command/
As said in https://github.com/openshift/openshift-docs/pull/20473#discussion_r394271218 , cluster admin can oc adm new-project openshift-xxx, which is CLI command.